halo2/src/circuit_builder.rs

```diff
@@ -166,6 +170,38 @@ pub(crate) fn analyzed_to_circuit<T: FieldElement>(
             _ => unimplemented!(),
         }
     }
+
+    assert!(
+        analyzed.public_declarations.len() <= num_rows,
+        "More publics than rows!"
+    );
+
+    // Enforce publics by copy-constraining to cells in the instance column.
+    let mut copys = vec![];
```

Comment:
```suggestion
    let mut copies = vec![];
```

Reply:
True! But funny that it's called `copys` in Plaf, so now we pass `copys: copies` :D